include/roboteam_ai/stp/Role.hpp

```diff
@@ -59,6 +59,11 @@ class Role {
      * @return true if the active tactic cannot execute (it's prerequisites are no longer met)
      */
     virtual bool shouldRoleReset(const StpInfo &info) noexcept = 0;
+
+    /**
+     * Current status of tactic state machine
+     */
+     Status currentTacticStatus;
```

Comment:
Why do you need to store the current tactic status as a member?

Reply:
I don't think I do haha, I forgot to remove this